site/instrument.ts

```diff
@@ -0,0 +1,35 @@
+import * as Sentry from "@sentry/react"
+import { isInIFrame } from "@ourworldindata/utils"
+import { COMMIT_SHA, ENV, SENTRY_DSN } from "../settings/clientSettings.js"
+import { getPreferenceValue, PreferenceType } from "./cookiePreferences.js"
+
+const analyticsConsent = getPreferenceValue(PreferenceType.Analytics)
+
+let sentryOpts: Sentry.BrowserOptions
+if (analyticsConsent && !isInIFrame()) {
+    // only collect session replays from: users that have consented to analytics
+    // AND where page isn't embedded in an iframe
+    sentryOpts = {
+        integrations: [
+            Sentry.replayIntegration({
+                maskAllText: false,
+                maskAllInputs: false,
+                blockAllMedia: false,
+                mask: [".sentry-mask"],
+            }),
+        ],
+        replaysSessionSampleRate: ENV === "development" ? 1 : 0.1,
+        replaysOnErrorSampleRate: 0,
+    }
+} else {
+    sentryOpts = {
+        replaysSessionSampleRate: 0,
```

Comment:
Do these need to be set at all if `Sentry.replayIntegration` isn't added?

Reply:
Not really, but I think Bobbie wanted to be more explicit so e.g. you don't have to look up the docs to know what's the default and I think it's fine to keep it as is.

Reply:
On the other hand, I think you are right, it makes less sense when the `replayIntegration` ins't added. Changing.